### PR TITLE
add no-join option

### PR DESCRIPTION
**Pull request by robertkowalski**
in order to make testing `/_setup_cluster` easier

### CODE REVIEW

File: dev/run [hunk 1]
```diff
@@ -248,6 +253,8 @@ def hashify(pwd, salt=COMMON_SALT, iterations=10, keylen=20):
 def startup(ctx):
     atexit.register(kill_processes, ctx)
     boot_nodes(ctx)
+    if ctx['no_join']:
+        return
```
Comment (kxepal) [inline dev/run]: I think need to make this check after `ensure_all_nodes_alive` - this step makes sure that nodes booted on previous step are ready for requests.

**Comment by robertkowalski** [comment]
fixed @kxepal - mind taking a second look?

**Comment by kxepal** [comment]
+1

**Comment by robertkowalski** [comment]
merged as 2a12d72

**Pull request closed without merging**